lib/libutils/isoc/include/sys/queue.h

```diff
@@ -229,6 +229,21 @@ struct {								\
 	}								\
 } while (/* CONSTCOND */0)
 
+#define	SLIST_REMOVE_SAFE(head, elm, type, field) do {			\
```

Comment:
I assume that you've invented this yourself because I can't find anything like this with google.
I have some doubts about adding this macro here. Generally when using these REMOVE macros you're supposed to know that the element is in the list. What does others think? @jforissier @etienne-lms @ruchi393

Reply:
I'd rather not add this macro here because <sys/queue.h> is a standard file, not a good place for customization. It seems it is used in only one place so what about `SLIST_FOREACH_SAFE() and `SLIST_REMOVE()` instead?

Reply:
Thanks @jforissier, I agree.

Reply:
Done